katalog/kibana/kibana.yml

```diff
@@ -24,9 +24,38 @@ spec:
       labels:
         app: kibana
     spec:
+      securityContext:
+        fsGroup: 1000
       containers:
       - name: kibana
-        image: docker.elastic.co/kibana/kibana:6.4.1
+        image: docker.elastic.co/kibana/kibana
+        securityContext:
+          capabilities:
+            drop:
+            - ALL
+          runAsNonRoot: true
+          runAsUser: 1000
+        readinessProbe:
+          exec:
+            command:
+              - sh
+              - -c
+              - |
+                #!/usr/bin/env bash -e
+                http () {
+                    local path="${1}"
+                    set -- -XGET -s --fail -L
+                    if [ -n "${ELASTICSEARCH_USERNAME}" ] && [ -n "${ELASTICSEARCH_PASSWORD}" ]; then
+                      set -- "$@" -u "${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD}"
+                    fi
+                    STATUS=$(curl --output /dev/null --write-out "%{http_code}" -k "$@" "http://localhost:5601${path}")
+                    if [[ "${STATUS}" -eq 200 ]]; then
+                      exit 0
+                    fi
+                    echo "Error: Got HTTP code ${STATUS} but expected a 200"
+                    exit 1
+                }
+                http /app/kibana
```

Comment:
😨

Reply:
I'm not a magician: https://github.com/elastic/helm-charts/blob/master/kibana/templates/deployment.yaml#L105